uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-request-popup-details-tab.component.spec.js

```diff
@@ -2014,6 +1506,8 @@ define([
     });
 
     /**
+<<<<<<< HEAD
```

Comment:
!!!!

Reply:
Oops, fixed. I ran unit tests before pushing, not sure how come it is still there. Thanks for noticing.